src/cli/commands/upgrade.js

```diff
@@ -11,6 +11,7 @@ export function setFlags(commander: Object) {
   // TODO: support some flags that install command has
   commander.usage('upgrade [flags]');
   commander.option('-S, --scope <scope>', 'upgrade packages under the specified scope');
+  commander.option('-L, --latest', 'upgrade packages to the latest version, ignoring version ranges in package.json');
```

Comment:
OK, was just following the convention of the other flags. I'll remove that `-L`.